server/src/main/resources/mime-type-catalog.csv

```diff
@@ -0,0 +1,989 @@
+.123,application/vnd.lotus-1-2-3
```

Comment:
Note to myself: Add support of RAW pictures extensions (CR2, CRW, SR2, SRF, etc)

https://stackoverflow.com/questions/43473056/which-mime-type-should-be-used-for-a-raw-image